query_execution/QueryManagerDistributed.cpp

```diff
@@ -119,6 +119,27 @@ bool QueryManagerDistributed::fetchNormalWorkOrders(const dag_node_index index)
   return generated_new_workorder_protos;
 }
 
+void QueryManagerDistributed::processInitiateRebuildResponseMessage(const dag_node_index op_index,
+                                                                    const std::size_t num_rebuild_work_orders) {
+  // TODO(zuyu): Multiple workers support.
```

Comment:
Yes, in the distributed version, without explicit explanations, `worker` means a worker process including `Shiftboss`, `StorageManager`, `CatalogDatabaseCache`, and a pool of `worker` threads.